.github/dependency-review-config.yml

```diff
@@ -0,0 +1,9 @@
+# For more details on the available options, see:
+# https://github.com/actions/dependency-review-action?tab=readme-ov-file#configuration-options
+fail-on-severity: critical
+
+comment-summary-in-pr: always
```

Comment:
We could switch this to `on-failure`, but I figured `always` would be nice to start with to see what this looks like in practice.

Reply:
Yeah, let's see what it looks like for now.